Installer/package.txt

```diff
@@ -367,6 +367,9 @@ component Documentation
 		rfolder "ide:Documentation/html_viewer/fonts"
 		rfolder "ide:Documentation/html_viewer/js"
 
+	into [[ToolsFolder]]/Documentation/html_viewer/resources/ place
```

Comment:
I think this should be `into [[ToolsFolder]]/Documentation/html_viewer/resources place` (i.e. without the `/`)